dask/sizeof.py

```diff
@@ -1,4 +1,6 @@
+import importlib.metadata
```

Comment:
Should we use the `importlib_metadata` backport package for earlier Python versions? If so, should we include that as a requirement on those Python versions?

Reply:
Python 3.8 includes `importlib.metadata`, so we only use the version guard to check which API to use (see next comment)